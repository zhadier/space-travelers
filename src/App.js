import { BrowserRouter, Routes, Route } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from './redux/configureStore';
import Header from './components/Header';
import BookListContainer from './components/BooklistContainer';
import Categories from './components/Categories';

const App = () => (
  <Provider store={store}>
    <BrowserRouter>
      <Header />
      <Routes>
        <Route path="/" element={<BookListContainer />} />
        <Route path="/categories" element={<Categories />} />
      </Routes>
    </BrowserRouter>
  </Provider>
);

export default App;
