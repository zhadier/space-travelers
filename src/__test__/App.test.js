import React from 'react';
import renderer from 'react-test-renderer';
import { cleanup } from '@testing-library/react';
import App from '../App';

afterEach(() => cleanup());

it('App Renders Correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
