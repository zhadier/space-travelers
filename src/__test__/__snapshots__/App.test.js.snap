// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders Correctly 1`] = `
Array [
  <header>
    <a
      aria-current="page"
      className="logo active"
      href="/"
      onClick={[Function]}
    >
      <img
        alt="logo"
        className="logo-img"
        src="planet-logo-2.svg"
      />
      <h1
        className="title"
      >
        Space Travelers' Hub
      </h1>
    </a>
    <nav
      className="header__nav"
    >
      <ul>
        <li>
          <a
            aria-current="page"
            className="active"
            href="/"
            onClick={[Function]}
          >
            Rockets
          </a>
        </li>
        <li>
          <a
            className=""
            href="/missions"
            onClick={[Function]}
          >
            Missions
          </a>
        </li>
        <li>
          <a
            className=""
            href="/profile"
            onClick={[Function]}
          >
            Profile
          </a>
        </li>
      </ul>
    </nav>
  </header>,
  <ul
    className="rockets-section"
  />,
]
`;
